src/components/DataClasses/route.js

```diff
@@ -0,0 +1,151 @@
+
+const wpRouteFromNoRoute = (noRoute) => {
+
+    // ----places/sites---
+    let PlaceSiteInWP=[]
+    noRoute.sites.map((PlaceSiteId) => {
```

Comment:
not critical - but map should be forEach
...
if you wanted to use map ... 
const PlaceSiteInWP =  noRoute.sites.map(PlaceSiteId => PlaceSiteId.id);